lib/pure/collections/sequtils.nim

```diff
@@ -634,7 +636,7 @@ since (1, 1):
     ## Returns a count of all the items that fulfilled the predicate.
     ##
     ## The predicate needs to be an expression using
-    ## the ``it`` variable for testing, like: ``countIt(@[1, 2, 3], it > 2)``.
+    ## the `it` variable for testing, like: `countIt(@[1, 2, 3], it > 2)`.
```

Comment:
can be deferred to future PR:
same comment as https://github.com/nim-lang/Nim/pull/16332/files#r541785498 for `numbers`